document-generator-api/src/main/java/uk/gov/companieshouse/document/generator/api/document/description/impl/RetrieveApiEnumerationDescriptionImpl.java

```diff
@@ -58,7 +57,7 @@ public String getApiEnumerationDescription(String fileName, String identifier, S
                     "descriptions for file name: " + descriptionsFile, e, setDebugMap(requestParameters));
         }
 
-        return populateParameters(description, parameters);
```

Comment:
If the expansion of description parameters is now handled by `company-accounts.api.ch.gov.uk` I would suggest refactoring this method and the interface that it satisfies to remove the fourth method parameter (named 'parameters' here) which is no longer used or required.